docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  db:
+    image: postgres:13.9
```

Comment:
It's better practice to just stick to a major version, ie `postgres:13`, as all binaries throughout the 13.x branch are guaranteed to be data/abi compatible, and the specific 13.9 minor version is not maintained nor receives any updates, with 13.14 being the current one right now.

(Also, while 13 will be supported until Sept 2025, it's most probably possible to bump to 16 without any schema changes or so).

Reply:
https://www.postgresql.org/support/versioning/